html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="">
+      <h1>Coding Form!</h1>
+      <div>
+        <label for="email">Email  <br/> <input type="text" name="email" id="email"></label>
+        </div>
+      <div>
+        Why is coding cool? <br/>
+        <label for="option">
+          <select name="option" id="option">
+            <option value="select">Select an option</option>
+            <option value="interesting">Because it's interesting</option>
+            <option value="fun">Because it's fun</option>
+            <option value="powerful">Because it's powerful</option>
+          </select>
+        </label>
+      </div>
+      <div>
+        Favorite Language (choose one) <br/>
+        <label for="language">
+          <input type="radio" name="language" value="HTML" id="HTML">HTML
```

Comment:
the only update would be to wrap the inputs with the labels here instead of using one label for all of them. And you don't need the id and for attributes for this special case!